packages/shared/src/components/modals/UserListModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useRef, useState } from 'react';
+import { UserShortInfoPlaceholder } from '../profile/UserShortInfoPlaceholder';
+import { Modal, ModalProps } from './common/Modal';
+import UserList, { UserListProps } from '../profile/UserList';
+import { InfiniteScrollingProps } from '../containers/InfiniteScrolling';
+import { UserShortProfile } from '../../lib/user';
+
+export interface UserListModalProps
+  extends Omit<ModalProps, 'children'>,
+    Pick<UserListProps, 'onOptionsClick'> {
+  users: UserShortProfile[];
+  placeholderAmount?: number;
+  title: string;
+  scrollingProps: Omit<InfiniteScrollingProps, 'children'>;
+}
+
+function UserListModal({
+  users,
+  title,
+  scrollingProps,
+  placeholderAmount,
+  onOptionsClick,
+  ...props
+}: UserListModalProps): ReactElement {
+  const container = useRef<HTMLElement>();
+  const [modalRef, setModalRef] = useState<HTMLElement>();
+
+  return (
+    <Modal
+      contentRef={(e) => setModalRef(e)}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Small}
```

Comment:
Also this is dynamic actually, currently the squad one is bigger then upvotes.
But you might want to check with design what's the idea there.

Reply:
Oh, the previous implementation had the props below. Let me check the accurate value in the Sketch then.
```
kind={Modal.Kind.FlexibleCenter}
size={Modal.Size.Small}
```

Reply:
Should now have the accurate value for size 🚢